tools/list_builder.py

```diff
@@ -3,27 +3,34 @@
 import copy
 import json
 import os
-import re
 import subprocess
-import sys
+import multiprocessing
+from pathlib import Path
 import time
+import shutil
 from collections import OrderedDict
-from pathlib import Path
-from shutil import which
-from typing import Any, Generator, TextIO
 
+import tqdm
+import logging
 import toml
 from git import Repo
 
+from app_caches import apps_cache_update_all, app_cache_folder  # pylint: disable=import-error
 from packaging_v2.convert_v1_manifest_to_v2_for_catalog import \
     convert_v1_manifest_to_v2_for_catalog  # pylint: disable=import-error
 
+from appslib.utils import (REPO_APPS_ROOT,  # pylint: disable=import-error
+                           get_catalog, git_repo_age)
+
+# Automatically enables error-to-xmpp
+import appslib.xmpplogger  # pylint: disable=import-error
```

Comment:
Kind of nitpicking but ideally we should pick a protocol-agnostic name for this like just "notifications" or idk ... The script is still called `sendxmpppy` but it's historical, under the hood it's actually calling matrix-commander :grimacing:

Reply:
okay :)

Reply:
Done: logging_sender